src/reference/02-DetailTopics/05-Plugins-and-Best-Practices/02-Plugins.md

```diff
@@ -508,6 +508,29 @@ libraryDependencies ++=
         Seq.empty
 ```
 
+### Publishing a plugin
+
+Sbt plugins can be published as any other projects. 
```

Comment:
```suggestion
sbt plugins can be published like any other projects. 
```

Reply:
For technical documentation like this, I think we should try to summarize the recommendation up front, assuming most people will skim through them.

```suggestion
Plugins can be published like any other projects. When publishing your plugin to a Maven-layout repository, use sbt 1.9.x or above.
```

Reply:
Good idea :+1: